pilot/cmd/pilot-agent/main.go

```diff
@@ -47,7 +47,7 @@ import (
 
 const (
 	localHostIPv4 = "127.0.0.1"
-	localHostIPv6 = "[::1]"
```

Comment:
If `[::1]` is not parsed properly somewhere, isn't that a bug? Its a valid IP, no?